x-pack/plugins/security_solution/server/usage/detections/ml_jobs/get_metrics.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  KibanaRequest,
+  SavedObjectsClientContract,
+  Logger,
+} from '../../../../../../../src/core/server';
```

Comment:
I changed it to `from 'kibana/server'` as that's where in other spots I was importing. Since it has the `type` at the front of it, `import type { KibanaRequest, SavedObjectsClientContract, Logger } from 'kibana/server';` that shouldn't slow anything down pulling it at a slighter higher level when it comes to tests.